xunit-performance.sln

```diff
@@ -0,0 +1,67 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 15
```

Comment:
I am not sure we want to add a solution file to the project.

Reply:
How do you normally open the code in Visual Studio?

Reply:
When you open a `csproj` file, VS automatically creates a `sln` file, and if needed I modify it locally. The reason I have not added it before is because with every VS version changes it modified the file too and I find it noisy.

Reply:
@brianrob Should we add a `sln` file?

Reply:
If you open a csproj file directly, VS doesn't automatically add any referenced projects to the solution it creates.  So if you opened up `scenariobenchmark.csproj`, you end up having to manually add `xunit.performance.api.csproj`, `xunit.performance.core.csproj`, `xunit.performance.execution.csproj`, and `xunit.performance.metrics.csproj` to the solution to get successful build / intellisense in VS.

I think almost all of the dotnet repos have `.sln` files checked in for this reason.

Reply:
I don't mind adding a solution file.  If we're going to, let's keep that change as a separate commit when the rest get squashed since it's unrelated to the rest of the changes.